src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1156,14 +1134,17 @@ PropertyPath Visitor::visit(Parser::PathNegatedPropertySetContext* ctx) {
 
 // ____________________________________________________________________________________
 PropertyPath Visitor::visit(Parser::PathOneInPropertySetContext*) {
-  // PathOneInPropertySet can only be called from PathNegatedPropertySet which
-  // itself is not supported.
+  // This rule is only used by the `PathNegatedPropertySet` rule which also is
+  // not supported and should already have thrown an exception.
   AD_FAIL();
 }
 
 // ____________________________________________________________________________________
-unsigned long long int Visitor::visit(Parser::IntegerContext* ctx) {
+uint64_t Visitor::visit(Parser::IntegerContext* ctx) {
   try {
+    // unsigned long long int might be larger than 8 bytes as per the standard.
+    // If that were the case this could lead to overflows.
```

Comment:
```suggestion
    // If that were the case this could lead to overflows.
    // TODO<joka921> Use `std::from_chars` but first check for the compiler support.
```

(I am sorry, I didn't understand your comment before reading your code, but you are completely right here).